tests/unittest/_torch/attention/test_flashinfer_attention.py

```
@@ -66,6 +66,25 @@ class CUDAGraphTestScenario:
 
 class TestFlashInferAttention(unittest.TestCase):
 
+    def test_attention_layer_indices_ignore_zero_kv_layers(self):
+
+        class FakeHybridManager:
+            layer_offsets = {0: 0, 1: 1, 2: 2}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)